composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": "^7.2",
-        "symfony/framework-bundle": "^3.4 || ^4.2",
+        "symfony/framework-bundle": "^3.4 || ^4.2 || ^5.0",
         "solarium/solarium": "^5.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^4.3"
+        "symfony/phpunit-bridge": "^4.3 || ^5.0"
```

Comment:
here we can actually only require `^5.0`

Reply:
To clarify why: `symfony/phpunit-bridge` is not listed within `symfony/symfony` and does not have any requirements to other packages. So we always just use the latest version here :blush:

Reply:
Done. Thanks for the information :-)